src/korean.js

```diff
@@ -93,6 +96,13 @@ class Korean {
     } // end of first else
   } // end of presentWord function
 
+  doPresentContinuous(word) {
+    let wordLength = word.length
+    let conjugate = word.split("").slice(0, wordLength-1)
```

Comment:
Korean word is a string of chars. There's no need to split it before slicing it.  And since it's just a string, you can easily concatenate strings by adding them. Please use the string '고있어'.